### PR TITLE
Starlight workshop

### PR DESCRIPTION
**Pull request by laulauland**
Adds a simple always-draft cheatsheet for using Starlight

### CODE REVIEW

**Comment by brettimus** [comment]
image

this looks a little clunky 👀

File: www/src/main.css [hunk 1]
```diff
@@ -124,3 +124,23 @@ h6 {
     text-decoration: underline;
   }
 }
+
+
+/* Styling buttons */
+
+kbd {
```
Comment (brettimus) [inline www/src/main.css]: might be good to put these styles on a kbd that has a particular classname? 

then you can revive the `KeyboardButton` component and just make sure it uses `<kbd>` under the hood to remain semantically apropos

Comment (brettimus) [reply www/src/main.css]: targeting `kbd` directly has the side effect of styling, for example, the keyboard shortcuts for search (i linked to an image in my comment on this pr)

Comment (laulauland) [reply www/src/main.css]: ahhh yea good catch

**Review by brettimus — APPROVED** [review]

**Pull request merged**